WeScan/Review/ReviewViewController.swift

```diff
@@ -9,11 +9,13 @@
 import UIKit
 
 /// The `ReviewViewController` offers an interface to review the image after it has been cropped and deskwed according to the passed in quadrilateral.
-final class ReviewViewController: UIViewController {
+open class ReviewViewController: UIViewController {
     
     private var rotationAngle = Measurement<UnitAngle>(value: 0, unit: .degrees)
-    private var enhancedImageIsAvailable = false
-    private var isCurrentlyDisplayingEnhancedImage = false
+    open var enhancedImageIsAvailable = false
+    open var isCurrentlyDisplayingEnhancedImage = false
+    open var enhanceButtonTintColorOn: UIColor = .white
+    open var enhanceButtonTintColorOff: UIColor = UIColor(red: 64 / 255, green: 159 / 255, blue: 255 / 255, alpha: 1.0)
```

Comment:
`: UIColor` is redundant here as we are already using a UIColor initializer. Can this be removed?